MQ2PluginManager.cpp

```diff
@@ -94,6 +100,78 @@ void DoPluginTool(PSPAWNINFO pChar, PCHAR szLine)
 	}
 }
 
+static MQPlugin* FindMQ2Plugin(const char* szLine)
+{
+	MQPlugin* pPlugin = pPlugins;
+	while (pPlugin)
+	{
+		if (!_stricmp(const_cast<char*>(szLine), pPlugin->szFilename))
+		{
+			return pPlugin;
+		}
+		pPlugin = pPlugin->pNext;
+	}
+	return nullptr;
+}
```

Comment:
GetPlugin exists and should be used instead of this.
https://github.com/macroquest/macroquest/blob/master/include/mq/api/PluginAPI.h#L137-L143

Also this const_cast is unnecessary.